mxfusion/components/distributions/gp/kernels/multiply_kernel.py

```diff
@@ -30,7 +30,7 @@ class MultiplyKernel(CombinationKernel):
     :param ctx: the mxnet context (default: None/current context).
     :type ctx: None or mxnet.cpu or mxnet.gpu
     """
-    def __init__(self, sub_kernels, name='add', dtype=None, ctx=None):
+    def __init__(self, sub_kernels, name='mul', dtype=None, ctx=None):
```

Comment:
I see this is the same change as in the request #197. It would be good to remove this change and other other changes that you make in #197 from this PR, and keep them contained to just that one. This helps keep both sets of reviews easier to do.

Also, I'll have another look at that PR, sorry I forgot about it recently. 

Thanks!